routes/campgrounds.js

```diff
@@ -1,8 +1,8 @@
-const   express = require("express");
-const   router = express.Router();
-        methodOverride = require("method-override"),
-        Campground = require("../models/campground"),
-        Comment = require("../models/comments");
+const express = require("express");
+const router = express.Router();
+methodOverride = require("method-override"),
```

Comment:
`methodOverride ` is not being used in this file

Reply:
Thats true i require it in vain, now im deleting it from that file